test/e2e-test/rule-test/config/testcases-cda.js

```diff
@@ -5,12 +5,28 @@
 
 // design the test cases
 var ruleSets = require('./rule-sets'); 
-var cdaTestcases = [
-    { dataFile: 'C-CDA_R2-1_CCD.xml.cda', templateFile: 'ccd.hbs', testRules: ruleSets.guidRules},
-    { dataFile: '170.314B2_Amb_CCD.cda', templateFile: 'ccd.hbs', testRules: ruleSets.commonRules},
-    { dataFile: 'Patient-1.cda', templateFile: 'ccd.hbs', testRules: ruleSets.commonRules},
-    { dataFile: 'sample.cda', templateFile: 'ccd.hbs', testRules: ruleSets.commonRules}
-];
+// var cdaTestcases = [
+//     { dataFile: 'C-CDA_R2-1_CCD.xml.cda', templateFile: 'ccd.hbs', testRules: ruleSets.guidRules},
+//     { dataFile: '170.314B2_Amb_CCD.cda', templateFile: 'ccd.hbs', testRules: ruleSets.commonRules},
+//     { dataFile: 'Patient-1.cda', templateFile: 'ccd.hbs', testRules: ruleSets.commonRules},
+//     { dataFile: 'sample.cda', templateFile: 'ccd.hbs', testRules: ruleSets.commonRules}
+// ];
+
+// module.exports = cdaTestcases;
```

Comment:
Could we remove the comments here? If we want to regress to the old version, we could find it in git commits.

Reply:
Sure, thay are all removed already.